types/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "werkit",
```

Comment:
It might be better to call this package `werkit-types`, as it doesn't have the same functionality as `werkit`.

Reply:
I was thinking that too, though I can imagine in the future there could be some common TypeScript functionality that we'd want to include here, as well. In that case it would be a good idea to secure the name.

If it turns out it does grow and we want to keep the types separate, we could publish `werkit-types` then.

Reply:
OK, sounds good.